__tests__/app.test.js

```diff
@@ -33,6 +33,37 @@ describe('GET Requests', () => {
                     expect(body).toEqual({});
                 });
         });
+        test('/api should provide a description of other endpoints that are available', () => {
+            return request(app)
+                .get('/api')
+                .expect(200)
+                .then(({ body }) => {
+                    const endpointKeys = Object.keys(body.endpoints);
+                    expect(endpointKeys.includes('GET /api')).toBe(true);
+                    expect(endpointKeys.includes('GET /api/topics')).toBe(true);
```

Comment:
This test would fail as we build the api and add more endpoints. In order to make this flexible, we can simply require in the endpoints.json file to the test file, then expect the response to equal the contents of it 👍

Reply:
Thanks, John.
I have refactored the test, hopefully it is ok. Do I just merge now or do I wait for further feedback?